bin/webpack.js

```diff
@@ -5,13 +5,30 @@
 	Author Tobias Koppers @sokra
 */
 var path = require('path');
+// var fs = require('fs');
 // Local version replace global one
 try {
 	var localWebpack = require.resolve(path.join(process.cwd(), 'node_modules', 'webpack-cli', 'bin', 'webpack.js'));
-	if(__filename !== localWebpack) {
+	if(localWebpack && path.relative(localWebpack, __filename) !== '') {
 		return require(localWebpack);
 	}
-} catch(e) {}
+} catch(e) {
+	/*
+	// Delete Cache, Retry and leave it if the path isn`t valid.
+	// swap with webpack-cli later
+	var dirPath = path.join(process.cwd(), 'node_modules', 'webpack', 'bin');
+	var retryLocal;
+	fs.readdir(dirPath, function(err, files) {
+		files.filter( (file) => {
+			delete require.cache[file];
+		});
+
+		// TODO: If path is wrong, leave it to higher powers until we publish on npm
+		// Could also install and load and install through github
+		return require(path.join(dirPath, 'webpack.js'))
+	});
+	*/
+}
```

Comment:
In development it goes to the catch block immediately, in production this would try to resolve first. Breaks now, as we don't have `node_modules/webpack-cli`